Apps/phvmray/vmray.json

```diff
@@ -1,37 +1,41 @@
 {
     "appid": "cfac072d-79fa-406b-a2e1-25a2fd5785c4",
     "name": "VMRay",
-    "description": "Connector for VMRay Analyzer",
-    "publisher": " VMRay ",
+    "description": "This app enables you to detonate files and URLs, and perform investigative actions, using the VMRay Platform, thereby giving you automated analysis and advanced threat detection through an agentless hypervisor-based sandbox",
+    "publisher": " VMRay",
+    "contributors": [
+        "Jens Thom"
+    ],
     "package_name": "phantom_vmray",
     "type": "sandbox",
-    "license": "Copyright (c) VMRay GmbH 2017",
-    "main_module": "vmray_connector.pyc",
-    "app_version": "1.0.4",
-    "min_phantom_version": "3.0.190",
+    "license": "Copyright (c) VMRay GmbH 2017-2021",
+    "main_module": "vmray_connector.py",
+    "app_version": "1.0.5",
```

Comment:
Please upgrade the major version as we've incorporated the changes for Python3 support; i.e. `2.0.0`